packages/xo-web/src/xo-app/vm/tab-backups.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'intl'
+import addSubscriptions from 'add-subscriptions'
+import constructQueryString from 'construct-query-string'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { some } from 'lodash'
+import { subscribeJobs } from 'xo'
+
+import LegacyOverview from '../backup/overview-legacy'
+import { JobsTable } from '../backup/overview'
+
+const ACTIONS = [
+  {
+    handler: (selectedJobs, { goTo }) =>
+      goTo({
+        pathname: '/backup/overview',
+        query: { s: `id: |(${selectedJobs.map(_ => _.id)})` },
+      }),
+    label: _('redirectToBackupJobs'),
+    icon: 'redirect',
+    individualLabel: _('redirectToBackupJob'),
+  },
+]
+
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: (job, { goTo }) =>
+      goTo({
+        pathname: '/home',
+        query: { t: 'VM', s: constructQueryString(job.vms) },
+      }),
+    label: _('redirectToMatchingVms'),
+    icon: 'preview',
+  },
+]
+
+const legacyJobKey = [
+  'continuousReplication',
+  'deltaBackup',
+  'disasterRecovery',
+  'backup',
+  'rollingSnapshot',
+]
+
+const BackupTab = decorate([
+  adminOnly,
+  addSubscriptions({
+    legacyJobs: subscribeJobs,
+  }),
+  provideState({
+    computed: {
+      haveLegacyBackups: (_, { legacyJobs }) =>
+        some(legacyJobs, job => legacyJobKey.includes(job.key)),
+      predicate: (_, { vm }) => ({ vms }) =>
+        vms !== undefined &&
```

Comment:
It not respect the specification. See https://github.com/vatesfr/xen-orchestra/pull/4860/files#diff-e20075cc846911786b066fc8090cdeabR156.